src/tests/unit/tests/common/components/cards/card-kebab-menu-button.test.tsx

```diff
@@ -166,16 +166,11 @@ describe('CardKebabMenuButtonTest', () => {
 
         rendered.find(ActionButton).simulate('click', event);
 
-        expect(rendered.state().showingCopyToast).toBe(false);
```

Comment:
Would be good to replace the removed assertions with new test content verifying the expected `show(...)` calls happen